src/core/transaction.test.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import moment from "moment";
+import Big from "big.js";
+
+import type {Transaction} from "./transaction";
+
+describe("transaction typechecking", () => {
+  it("can construct transaction for various transaction types", () => {
+    // Not intended to be exhaustive, just verifying that the element types work
```

Comment:
This is fine, but it _would_ be easy enough to be exhaustive with
`Object.keys`.

Reply:
I kind of like the explicit construction.

Reply:
Go for it, then. :-)